### PR TITLE
Fix a typo in FeePayer

### PR DESCRIPTION
**Pull request by nicholaspai**
Signed-off-by: Nick Pai <npai.nyc@gmail.com>

### CODE REVIEW

File: core/contracts/financial-templates/implementation/FeePayer.sol [hunk 1]
```diff
@@ -120,13 +120,13 @@ abstract contract FeePayer is Testable, Lockable {
         // If the effective fees paid as a % of the pfc is > 100%, then we need to reduce it and make the contract pay as much of the fee
         // that it can (up to 100% of its pfc). We'll reduce the late penalty first and then the regular fee, which has the effect of paying
         // the store first, followed by the caller if there is any fee remaining.
-        if (totalPaid.isGreaterThan(_pfc)) {
```
Comment (mrice32) [inline core/contracts/financial-templates/implementation/FeePayer.sol:123]: How did this pass ci?

Comment (nicholaspai) [reply core/contracts/financial-templates/implementation/FeePayer.sol:123]: I have no idea? it failed immediately post-merge?

Comment (nicholaspai) [reply core/contracts/financial-templates/implementation/FeePayer.sol:123]: merge conflicts somehow i'm guessing

Comment (chrismaree) [reply core/contracts/financial-templates/implementation/FeePayer.sol:123]: how are we having things pass CI that we think should fail? it's a bit worrying.

Comment (nicholaspai) [reply core/contracts/financial-templates/implementation/FeePayer.sol:123]: Its happening when PR 1 is approved, and then another PR 2 is merged, and then the PR 1 is merged into master, which github auto-resolves any merge conflicts between PR 1 and 2.

Perhaps we should force manual merges with master?

**Pull request merged**